docs/themes-list/powerline-base.rst

```diff
@@ -70,6 +70,7 @@ The contents of the prompt can be "reordered", all the "segments" (every piece o
 * ``battery`` - Battery information (you'll need to enable the ``battery`` plugin)
 * ``clock`` - Current time in ``HH:MM:SS`` format
 * ``cwd`` - Current working directory including full folder hierarchy (c.f. ``wd``\ )
+* ``duration`` - Duration of the last command. See :ref:`Command duration <command_duration>` for details.
```

Comment:
Yes, I have built the documentation locally and this points to the correct page.
In the live docs it would point to https://bash-it.readthedocs.io/en/latest/themes/#command-duration.

Reply:
>  I have built the documentation locally

Way to go over and above - thanks for confirming !